### DISCUSSION TITLE
[Feature] Code view

### DISCUSSION THREAD

**Discussion by Tschipp** [bridge-core/bridge.#299] [Ideas]
**Summary**
I would really appreciate a code view for files, where I can directly see and edit the raw JSON. 

**Additional context/Motivation**
I find myself wanting to switch to something like VSCode when changing a lot of stuff or copying things from existing entities (which aren't necessarily in the same project). But since that messes with bridge's cache, that is almost never a good idea. Of course, reason for not implementing this is because you want to prevent people from writing erroneous JSON files, which I understand. I just think it is a bit cumbersome sometimes, especially when I'm copying things from other previous works.

**Alternatives/Considerations**
I can't really think of any alternatives.

**Comment by Chikorita-Lover** [comment]
This is already planned for version 2.0.0

**Comment by solvedDev** [comment]
Thanks for the suggestion! We are going to go text editing first for bridge v2 which is in the works right now 👍

**Comment by jasonjgardner** [comment]
![Extension in bridge. v2-rc3](https://user-images.githubusercontent.com/1903667/123500590-c9df4d00-d604-11eb-9684-fb16be4f4745.png)

[I've started working on a bridge. v2 extension which can send project files to VS Code.](https://github.com/jasonjgardner/bridge-plugins/tree/999f310e29104f01ebdb19ac3b5e6c0582c4ed83/plugins/OpenInVSCode)

The user needs to provide the path to the bridge. installation to enable a project file search input. If the installation path provided _appears_ to be a valid, it is saved to bridge's `data/settings.json` file as the `openInVSCode.realPath` property.

In theory, this could be used to send a project file path to _any_ URL protocol handler.
